Tasks/ANT/package.json

```diff
@@ -1,12 +1,14 @@
 {
-  "name": "Ant",
+  "name": "Agent.Tasks.Ant",
+  "version": "1.0.52",
```

Comment:
remove this from the package.json.  It's not needed and it will just get out of sync with what's in the task.json.